styles/style.css

```diff
@@ -0,0 +1,491 @@
+
+header {
+  border-bottom: 1px solid lightgray;
+  padding-bottom: 5px;
+  overflow-x: hidden;
+}
+
+nav {
+    position: fixed;
```

Comment:
Consider adding a `background-color` and `z-index` property to this element. (See illustration below)
- a white `background-color` will change the transparent display to white and
- `z-index` of any integer value will prevent other fixed positions from covering the navbar since the navbar should be the one always on top. See [this to learn more about `z-index`](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index)

![demo-nav-nyt](https://user-images.githubusercontent.com/36057474/65393635-f81e6180-dd7a-11e9-8e4c-effa566a453e.gif)

```suggestion
    position: fixed;
    background-color: white;
    z-index: 1;
```